features/api/plan_features.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Plan Features
+  In order to list which features of my API are available for each plan
+  As a provider
+  I want to define features
+
+  Background:
+    Given a provider "foo.3scale.localhost"
+    And an application plan "Basic" of provider "foo.3scale.localhost"
+    Given current domain is the admin domain of provider "foo.3scale.localhost"
+
+  @javascript
+  Scenario: Create a feature
+    When I log in as provider "foo.3scale.localhost"
+    And I go to the edit page for plan "Basic"
+    And I follow "New feature"
+    And I fill in "Name" with "Free T-shirt"
+    And I fill in "System name" with "tee"
+    And I fill in "Description" with "T-shirt with logo of our company for free."
+    And I press "Save"
+    Then I should see "T-shirt with logo of our company for free."
+    And I should see the flash message "Feature has been created."
+
+  @javascript
+  Scenario: "No features yet" notice disappears when the first feature is created, and appears when last one is deleted.
+    When I log in as provider "foo.3scale.localhost"
+    And I go to the edit page for plan "Basic"
+    Then I should see "This plan has no features yet."
+
+    When application plan "Basic" has "Some Feature" enabled
+    And I go to the edit page for plan "Basic"
+    Then I should not see "This plan has no features yet."
+
+    When application plan "Basic" does not have any features
+    And I go to the edit page for plan "Basic"
+    Then I should see "This plan has no features yet."
```

Comment:
Given, when, then is supposed to be used once not thrice. Spaces also suggests this Scenario should split.

Reply:
Ugh... then it would be just too much repetition... :unamused: 
I'll try smth...

Reply:
Done